sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -576,7 +576,7 @@ void CurlSession::ParseChunkSize(Context const& context)
   // Move to after chunk size
   for (bool keepPolling = true; keepPolling;)
   {
-    for (int64_t index = this->m_bodyStartInBuffer, i = 0; index < this->m_innerBufferSize;
+    for (int32_t index = this->m_bodyStartInBuffer, i = 0; index < this->m_innerBufferSize;
          index++, i++)
```

Comment:
Mixing `index` and `i` probably doesn't read very well. Rename `i` to `iteration` and `index` to `offset`?